### PR TITLE
Release v1.108.0 - release -> `staging`

### PR DESCRIPTION
**Pull request by jdamore-linode**
## Description 📝
Merges the 1.108.0 release branch to `staging`.

## How to test 🧪
Once the preview link has been generated, we can test manually, or we can run the Cypress tests against the preview link. This can be done (I think) by checking out the release branch, running `yarn && yarn build`, and then running:

```bash
CYPRESS_BASE_URL="<preview link here>" yarn cy:run
```

### CODE REVIEW

**Review by abailly-akamai — DISMISSED** [review]
Spent quite a bit of time with just the code for this release and spotted a couple non important items - otherwise looks pretty solid at a glance (it's a big one!) - Love to see the amount of coverage (both e2e and unit) added as part of this end of year release 🎉

File: packages/manager/cypress/e2e/core/images/create-linode-from-image.spec.ts [hunk 1]
```diff
@@ -55,6 +55,8 @@ const createLinodeWithImageMock = (url: string, preselectedImage: boolean) => {
 
   cy.wait('@mockLinodeRequest');
 
+  console.log('mockLinode', mockLinode);
```
[the remark this answers is not in the document]
Comment (jdamore-linode) [reply packages/manager/cypress/e2e/core/images/create-linode-from-image.spec.ts:58]: @abailly-akamai Agreed! I know this isn't the only stray call to `console.log` in the test files, so I opened a ticket in the "Cypress Clean-Up and Refactoring" epic to clean them up. Thanks for the catch!

**Review by cpathipa — APPROVED** [review]

File: packages/manager/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,95 @@ All notable changes to this project will be documented in this file.
...
+- Improve MainConcept Transcoders Marketplace app name, description, and website ([#9858](https://github.com/linode/manager/pull/9858))
+- Move Linode Details Add/Edit Config button alignment to the right ([#9925](https://github.com/linode/manager/pull/9925))
+- Add pricing Docs Link to create/clone flows and remove DC-specific pricing warning notice  ([#9946](https://github.com/linode/manager/pull/9946))
+- Update MainConcept app names to include “Demo” ([#9950](https://github.com/linode/manager/pull/9950))
+
+
+### Fixed:
+
+- Overflow for VPC and StackScript detail descriptions and cut off placeholder text in VPC search bar ([#9887](https://github.com/linode/manager/pull/9887))
+- Add missing region ID param to Linode Detail clone action menu item ([#9888](https://github.com/linode/manager/pull/9888))
```
Comment (mjac0bs) [inline packages/manager/CHANGELOG.md:26]:
```suggestion
- Missing region ID param in Linode Detail clone action menu item ([#9888](https://github.com/linode/manager/pull/9888))
```
Ugh, missed this. Very minor, I'll live if we don't fix it.

Comment (jdamore-linode) [reply packages/manager/CHANGELOG.md:26]: I'll fix this before the merge 👍 holding off for now just to avoid interfering with Dajahi's work

**Review by bnussman-akamai — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

**Review by abailly-akamai — APPROVED** [review]

**Review by mjac0bs — APPROVED** [review]
My browser can barely load this diff. 😅 

Changes look good from release testing, fixes are in, and confirmed the changelogs and package version bumps.

**Comment by jdamore-linode** [comment]
Merging despite checks failure. The only failed check (Code Coverage) was expected because the feature doesn't exist in `staging` yet -- it's expected to work for subsequent releases. All other checks passed ✅

**Pull request merged**